packages/jest-util/src/deepCyclicCopy.ts

```diff
@@ -7,6 +7,11 @@
 
 const EMPTY = new Set<string>();
 
+const isDate = (value: any): value is Date => value.constructor === Date;
```

Comment:
Should we use `jest-get-type` instead?

Reply:
If we use `jest-get-type`, it doesn't actually give type information to typescript compiler.
It need to do extra double type conversion when passing variable to function.

```typescript
const type = getType(value);
if (type === 'array') {
  functionPassingArray((value as unknow) as Array<T>);
}
```

Reply:
Ah, ok. Let's keep it as is, then 🙂 We could potentially export the testers from get-type to reduce the duplication, but I think this is fine as-is